core/trino-main/src/test/java/io/trino/cost/TestFilterStatsCalculator.java

```diff
@@ -301,6 +311,92 @@ public void testAndStats()
 
         assertExpression("CAST(NULL AS boolean) AND CAST(NULL AS boolean)").equalTo(zeroStatistics);
         assertExpression("CAST(NULL AS boolean) AND (x < 0e0 AND x > 1e0)").equalTo(zeroStatistics);
+
+        Consumer<SymbolStatsAssertion> symbolAssertX = symbolAssert -> symbolAssert.averageRowSize(4.0)
```

Comment:
nit: move `.averageRowSize(4.0)` to next line

Reply:
no, it looks to me better this way and there are other examples of similar formatting in this file